### PR TITLE
Rearranged JDBC buttons

### PR DESCRIPTION
**Pull request by Atlas139mkm**
### Description of the Change

- Changed "cancel" - "Ok
- Moved "Remove" to left hand side

Drivers Tab
image

- Changed "cancel" - "Ok
- Moved "Remove" to left hand side (where modify was)
- Moved "Modify" to the right of "Remove"

Connections Tab
image

- Changed "Test" - "Test Connection" to be more self explanatory of what it does.

Add/Modify connections screen
image

### Why Should This Be In Core?

The screen flows nicer and buttons are more intuitive to what they would be doing.

### Benefits

UX UI

### Possible Drawbacks

I imagine none

### Verification Process

All buttons still work as intended, only visual text was changed

### Applicable Issues

#1322

### CODE REVIEW

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java [hunk 1]
```diff
@@ -167,9 +172,10 @@ public JDBCSourcePane(final JDBCImportController importController) {
                     dbConnectionComboBox.setItems(connections);
                 }
             });
-            connectionsPane.add(removeBtn, 2, 1, 1, 1);
+            addRemModGp.add(removeBtn, 1, 1, 1, 1);
+            connectionsPane.add(addRemModGp, 0, 1, 1, 1);
 
-            final Button buttonCancel2 = new Button(ACTION_CANCEL);
+            final Button buttonCancel2 = new Button("Ok");
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java:178]: Can you make Ok upper case? ("OK")

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java [hunk 2]
```diff
@@ -309,9 +315,9 @@ public JDBCSourcePane(final JDBCImportController importController) {
 
             final Button removeBtn1 = new Button("Remove");
             removeBtn1.setOnAction(removebtnAction);
-            driversTabGridPane.add(removeBtn1, 2, 1, 1, 1);
+            driversTabGridPane.add(removeBtn1, 1, 1, 1, 1);
 
-            final Button buttonCancel = new Button(ACTION_CANCEL);
+            final Button buttonCancel = new Button("Ok");
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java:320]: similar

**Review by antares1470 — CHANGES_REQUESTED** [review]
If you could change the order of the buttons on the connections tab to Add, Modify, Remove as well that would be great. I think that is a more logical flow compared to Add, Remove, Modify

**Comment by Atlas139mkm** [comment]
@antares1470 I agree, I did originally want to do that. I had a reason for not however. Swapping between the 2 tabs would then rearrange the buttons. 

drivers tab:
Add Remove

Connections tab:
Add Remove Modify - adds modify to the end

The other way:

drivers tab:
Add Remove

Connections tab:
Add **Modify** Remove  - rearranges the buttons

Let me know your thoughts, I can change it if you still would prefer it.

**Comment by antares1470** [comment]
@Atlas139mkm I rearranging the buttons will be the way to go. Unless there end up being some serious performance issues with doing so, the logical flow is still the preferred.

**Review by antares1470 — APPROVED** [review]

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java [hunk 3]
```diff
@@ -167,9 +172,10 @@ public JDBCSourcePane(final JDBCImportController importController) {
                     dbConnectionComboBox.setItems(connections);
                 }
             });
-            connectionsPane.add(removeBtn, 2, 1, 1, 1);
+            addRemModGp.add(removeBtn, 2, 1, 1, 1);
+            connectionsPane.add(addRemModGp, 0, 1, 1, 1);
 
-            final Button buttonCancel2 = new Button(ACTION_CANCEL);
+            final Button buttonCancel2 = new Button("OK");
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java:178]: actually, can you rename the variable here sine it is no longer cancel?

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java [hunk 4]
```diff
@@ -309,9 +315,9 @@ public JDBCSourcePane(final JDBCImportController importController) {
 
             final Button removeBtn1 = new Button("Remove");
             removeBtn1.setOnAction(removebtnAction);
-            driversTabGridPane.add(removeBtn1, 2, 1, 1, 1);
+            driversTabGridPane.add(removeBtn1, 1, 1, 1, 1);
 
-            final Button buttonCancel = new Button(ACTION_CANCEL);
+            final Button buttonCancel = new Button("OK");
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java:320]: same here

**Review by antares1470 — CHANGES_REQUESTED** [review]

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java (hunk 3 above)
Comment (arcturus2) [reply CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java:178]: @Atlas139mkm can you rename this to `buttonOk2` or something better?

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java (hunk 4 above)
Comment (arcturus2) [reply CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java:320]: @Atlas139mkm can you rename this to `buttonOk` or something better?

**Review by arcturus2 — CHANGES_REQUESTED** [review]
Some minor feedback left for you regarding variable names

**Comment by Atlas139mkm** [comment]
yep can do, I believe I just left them with their existing names and changed the text. But I can change the variable names to match.

**Review by arcturus2 — APPROVED** [review]
Thanks.

**Pull request merged**